File: API/XApp6/XApp6/XApp6/Models/Documento.cs
using System;
namespace XApp6.Models
{
    public class Documento
    {
		public int id { get; set; }
		public string nome { get; set; }
        public string categoria { get; set; }
		public int age { get; set; }
        public List<Curso> cursos { get; set; }
    }
}

File: finalbackoffice/backoffice/app/Documento.php
<?php

namespace App;

use Illuminate\Database\Eloquent\Model;

class Documento extends Model
{
	protected $table = "documentos";
	protected $primaryKey = "id";
	protected $fillable = array("nome", "categoria");
	public $timestamps = true;
	
	public function cursos()
	{
		return $this->belongsToMany('App\Curso', 'curso_documento')->withTimestamps();
	}
}